js/api-augment/README.md

```diff
@@ -49,12 +49,12 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
 
 ### Updating Definitions
 
-1. Update `./definitions`
```

Comment:
@dopatraman  I was about review 

But you should not delete point 1 , it is important as it is a manual step 

Please fix this in some other PR